src/servers/kfserving_http_server.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Just use this same header for both V1 and V2 and since using the TRTIS_ENABLE_HTTP_V2 flag to build one source file or the other will get the correct implementation.

Reply:
Done